### DISCUSSION TITLE
The function "disable thumbnails" doesn't work.

### DISCUSSION THREAD

**Discussion by niubility000** [filebrowser/filebrowser#1198] [Q&A] [answered]
**Description**
Version 2.10.0 works great when I use my computer as the server. But if I use my router (cpu: BCM4709 Cortex a9 1.4GHz) as the server, the filebrowser always crashes when I browse a folder with around 1000 pictures. 
Version 2.2.0 can run steadily on my router. Filebrowser after 2.2.0 began to resize images. I think the router’s cpu is not powerful enough to handle that. I tried the new functions in version 2.10.0: to disable thumbnail, to stop resizing images, to enable cache, and to limit the number of image processors. However, all of them do not work in version 2.10.0. I used the commands like: filebrowser -d filebrowser.db config set --disable-preview-resize, or --img-processors 1, or --disable-thumbnails, or --cache-dir cache. I exported the config file. It showed “enable thumbnails”: false. “enable preview resize”: false. But when I browsed the web page, the thumbnails still showed up, and the filebowser service on my router would crash finally(showed: Something really went wrong.)
And also, when I browsed the webpage from my route server, the images didn’t show up in my android phone’s browser. I had to switch the HD mode to the dropdown mode every time to show the pictures. It's not convenient. So, please add a command to set the “dropdown” mode as the default value.

**Expected behaviour**
I hope version 2.10.0 can works as steady as version 2.2.0 on my router. I don’t need “thumbnail” function on my router totally. I think a lot of people use routers as servers.

**Comment by niubility000** [comment]
By the way, I use the linux arm v5 version on my router.

**Comment by o1egl** [comment, marked as answer]
You are using cli params in a wrong way. It should look like `filebowser -d filebowser.db --disable-preview-resize --disable-thumbnails`

**Reply by niubility000** [reply]
Oh，thank you so much to correct it! I will try it now.

Merry Christmas and happy new year!

**Reply by niubility000** [reply]
I found the reason why the service shut down sometimes. It's "out of memory".  My router's CPU is good enough. My router's memory is 500MB. So I added 1GB virtual memory to it using my hard disk. I just rewrote the file "services-start" : 
#!/bin/sh
cd /tmp/mnt/h1backup/filebrowser
sleep 15
chmod 755 filebrowser
sleep 5
str=$"/n"
sstr=$(echo -e $str)
nohup /tmp/mnt/h1backup/filebrowser/filebrowser -d filebrowser.db --disable-preview-resize --cache-dir /tmp/mnt/h1backup/filebrowser/cache >/dev/null 2>&1 &
echo "$sstr"

Now, it works perfect!
o1egl, thank you so much!
